mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1152,6 +1153,9 @@ def LLVM_AddressOfOp : LLVM_Op<"mlir.addressof",
   let hasFolder = 1;
 }
 
+def DIGlobalVariableExpressionArrayAttr :
+  TypedArrayAttrBase<LLVM_DIGlobalVariableExpressionAttr, "an array of variable expressions">;
```

Comment:
I would consider moving this right below the definition of LLVM_DIGlobalVariableExpressionAttr? We already have such arrays for example for LLVM_AliasScopeArrayAttr.

Reply:
Done.